dracut/modules.d/99kiwi-lib/kiwi-luks-lib.sh

```diff
@@ -39,7 +39,10 @@ function activate_luks {
         luksOpen "${device}" luks
     else
         # There is a keyfile and we need to get prompted to enter the passphrase
-        /usr/lib/systemd/systemd-cryptsetup attach luks "${device}"
+        /usr/lib/systemd/system-generators/systemd-cryptsetup-generator \
```

Comment:
The unit should already exist, so this call shouldn't be necessary

Reply:
yes and you are absolutely sure that between the first generator call and the time we resize that there is no combustion, ignition, some-other-magic that kicks something in the background and changes ```/etc/crypttab``` ? If you say yes, I delete it :-)

Reply:
ok good point, I'll drop it